src/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -540,14 +540,17 @@ public async Task SendDefaultResponseAsync(int streamId)
             await WriteFrameAsync(headersFrame).ConfigureAwait(false);
         }
 
-        public async Task SendResponseHeadersAsync(int streamId, bool endStream = true, HttpStatusCode statusCode = HttpStatusCode.OK, IList<HttpHeaderData> headers = null)
+        public async Task SendResponseHeadersAsync(int streamId, bool endStream = true, HttpStatusCode statusCode = HttpStatusCode.OK, bool hasStatusCode = true, IList<HttpHeaderData> headers = null)
```

Comment:
Does `hasStatusCode` have a useage outside of trailing headers? If not, I would consider renaming this to something like `isTrailingHeader`.